src/main/java/V.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class V {
+
+    private static final String LINE_SEPERATOR = "____________________________________________________________";
+
+    // Print 2 line seperators between a block of text for cleaner CLI
+    public static void printBlock(String text) {
+        System.out.println(LINE_SEPERATOR);
+        System.out.println(text);
+        System.out.println(LINE_SEPERATOR);
+    }
+
+    public static void greet() {
+        String logo = " _       _ \n"
+                + "\\ \\     / / \n"
+                + " \\ \\   / / \n"
+                + "  \\ \\_/ / \n"
+                + "   \\___/ \n";
+        System.out.print(logo);
+        printBlock("Hi I'm V\nWhat can I do for you?");
+    }
+
+    public static void displayList(Task[] listOfTasks, int length) {
+        System.out.println(LINE_SEPERATOR);
+        for (int i = 0; i < length; i++) {
+            System.out.println(String.format("%d.[%s]%s", i + 1,  
+                    listOfTasks[i].getType(), listOfTasks[i]));
+        }
+        System.out.println(LINE_SEPERATOR);
+    }
+
+    public static void main(String[] args) {
+        greet();
+
+        boolean isOnline = true;
+        Task[] listOfTasks = new Task[100];
+        int count = 0;
+        String temp;
+        String description;
+        String by;
+        String[] timePeriod;
+        String from;
+        String to;
+        String line;
+        String[] lineArr;
+        Scanner input = new Scanner(System.in);
+        
+        while (isOnline) {
+            line = input.nextLine();
+            lineArr = line.trim().split(" ");
+            switch (lineArr[0]) {
+            case "bye":
+                input.close();
+                isOnline = false;
+                break;
```

Comment:
Good use of boolean flag isOnline as it allows the program to have a clear flow and makes the code easy to understand.